utils.go

```diff
@@ -254,5 +254,18 @@ func ForceSymlink(orig, symlink string) error {
 	if err != nil {
 		return errw.Wrap(err, "symlinking file")
 	}
-	return nil
+
+	return SyncFS(symlink)
+}
+
+func SyncFS(syncPath string) (errRet error) {
```

Comment:
👍  

Not for this PR but it feels like we should move this into a go-utils so both the agent and viam-server use the same code.